Hunting Queries/AzureDevOps/AAD Conditional Access Disabled.yaml

```diff
@@ -0,0 +1,19 @@
+id: c78a3845-37d9-448e-a8cd-e9543f00bcc5
+name: Azure DevOps- AAD Conditional Access Disabled 
+description: |
+  'This hunting query identifies Azure DevOps activities where organization AADConditionalAccess policy disable by the admin'
+tactics:
+  - Persistence
+  - DefenseEvasion
+relevantTechniques:
+  - T1098
+  - T1089
+query: |
+  let timeframe = 7d;
+  AzureDevOpsAuditing
+  | where TimeGenerated >= ago(timeframe)
+  | where OperationName =="OrganizationPolicy.PolicyValueUpdated"
+  | where Data.PolicyName == "Policy.EnforceAADConditionalAccess"
+  | where Data.PolicyValue == "OFF"
+  | extend TimestampCustomEntity = TimeGenerated, AccountCustomEntity = ActorUPN, IPCustomEntity = IpAddress, URLCustomEntity = ScopeDisplayName
```

Comment:
Instead of   TimestampCustomEntity  we should use timestamp. See more details here for entity mapping:
https://github.com/Azure/Azure-Sentinel/wiki/Query-Style-Guide#entity-mapping

Reply:
Thanks @aprakash13 I pushed an updated to accommodate this